test/jit_utils.py

```diff
@@ -579,3 +579,27 @@ def wrapper(*args, **kwargs):
                 return fn(*args, **kwargs)
             return wrapper
         return noop_fuser
+
+
+class Observer(torch.nn.Module):
```

Comment:
I'm not sure about this - it doesn't seem as generic as everything else in this file. It's really a very test specific (even though we have several tests using it). Ideally I think that should live in some other file along with the tests for JIT passes for quantization. If other people think it's fine, I don't mind though.

Reply:
put them in JitTestCase?